recipe/patches/0002-run-host-H5detect-when-cross-compiling.patch

```diff
@@ -1,20 +1,19 @@
-From 6e84837b5865ee12196a6de7955e16324ffa7d7c Mon Sep 17 00:00:00 2001
+From c3b21fe8632a8d03525b07ea671ebdb44aeb5cbd Mon Sep 17 00:00:00 2001
 From: Min RK <benjaminrk@gmail.com>
 Date: Thu, 15 Jun 2023 00:06:44 +0200
-Subject: [PATCH] run host H5detect when cross-compiling
+Subject: [PATCH 2/2] run host H5detect when cross-compiling
 
 ---
  configure.ac               | 8 ++++++++
  fortran/src/Makefile.am    | 4 ++--
  hl/fortran/src/Makefile.am | 2 +-
- src/Makefile.am            | 4 ++--
- 4 files changed, 13 insertions(+), 5 deletions(-)
+ 3 files changed, 11 insertions(+), 3 deletions(-)
 
 diff --git a/configure.ac b/configure.ac
-index 6e9421592f..598e8b6b64 100644
+index 830dd176da..a1fe555a21 100644
 --- a/configure.ac
 +++ b/configure.ac
-@@ -129,6 +129,14 @@ AC_SUBST([AR_FLAGS])
+@@ -140,6 +140,14 @@ AC_SUBST([AR_FLAGS])
  
  AC_SUBST([cross_compiling])
  
```

Comment:
H5detect no longer exists, so you might want to modify the comments below to indicate that src_run is just needed for running Fortran tests.

Reply:
ok thanks.